### DISCUSSION TITLE
MILESTONE NOTIFICATION

### DISCUSSION THREAD

**Discussion by s10singh97** [jtr13/cc21fall1#85] [General]
Hey guys,

Great work on maintaining and merging the PRs. I just wanted to let you all know that we have already crossed the 50% PR Merges Milestone. Congratulations to everyone for working on these PRs! 💯 🎉 

BTW :heavy_check_mark: [CLICK HERE](https://ibb.co/MNHmNbk) to find out Who's Awesome?

We just have 22 more open PRs. Let us all try to merge and close all the PRs in the next couple of days!

As always, please feel free to tag me in case you need any help!

Thanks!

**Comment by jtr13** [comment]
Great job everyone and great post @s10singh97 !!
